src/components/ProductForm/DeductiblesCeilingsTabForm.js

```diff
@@ -177,12 +182,18 @@ const DeductiblesCeilingsTabForm = (props) => {
             <TableRow className={classes.tableTitle}>
               <TableCell width="200" />
               {!isSplit && <TableCell>{formatMessage("DeductiblesCeilingsTabForm.all")}</TableCell>}
-              {isSplit && (
+              {isSplit && edited.ceilingInterpretation == "HEALTH_FACILITY_TYPE" && (
                 <>
                   <TableCell>{formatMessage("DeductiblesCeilingsTabForm.hospitals")}</TableCell>
                   <TableCell>{formatMessage("DeductiblesCeilingsTabForm.non_hospitals")}</TableCell>
                 </>
               )}
+              {isSplit && edited.ceilingInterpretation == "CLAIM_TYPE" && (
```

Comment:
Use '===' instead of '=='. Moreover, this "CLAIM_TYPE" should be moved to the constants.

Reply:
done